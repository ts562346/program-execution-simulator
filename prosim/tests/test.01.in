test1 2
BLOCK 3
HALT

