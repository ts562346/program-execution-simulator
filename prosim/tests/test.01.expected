00000: BLOCK
00003: HALT

DOOP count : 0
DOOP time  : 0
BLOCK count: 1
BLOCK time : 3
